roachtest: gopg failed

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.gopg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18520368?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18520368?buildTab=artifacts#/gopg) on release-25.1 @ [0518bc00124b6f9f5351fdfa5fe0b85af345ec17](https://github.com/cockroachdb/cockroach/commits/0518bc00124b6f9f5351fdfa5fe0b85af345ec17):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.0-beta.1-dev-0518bc00124b6f9f5351fdfa5fe0b85af345ec17
Tests run against gopg v10.9.0
193 Total Tests Run
178 tests passed
15 tests failed
0 tests skipped
6 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: v10.TestColumnReuse - unknown (unexpected)
For a full summary look at the gopg artifacts 
An updated blocklist (gopgBlockList) is available in the artifacts' gopg log
(cluster.go:2478).Run: context canceled
test artifacts and logs in: /artifacts/gopg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18520368/gopg/1737024054725/1737024352030)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gopg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-46557

--- a/pkg/cmd/roachtest/tests/gopg_blocklist.go
+++ b/pkg/cmd/roachtest/tests/gopg_blocklist.go
@@ -50,4 +50,6 @@ var gopgIgnoreList = blocklist{
 	"pg | DB race | SelectOrInsert with OnConflict is race free":    "unknown",
 	"pg | DB race | SelectOrInsert without OnConflict is race free": "unknown",
 	`pg | ORM | relation with no results does not panic`:            "unknown",
+	// This test flakes sometimes because of connection reuse.
+	`v10.TestColumnReuse`: "unknown",
 }

